src/environments/environment.prod.ts

```diff
@@ -2,4 +2,6 @@ export const environment = {
   production: true,
   API_URL: 'https://api.drecs.org/api/',
   Explorer_URL: 'https://explorer.energyweb.org',
+  SENTRY_DNS:
```

Comment:
can't we pass environment variables to the app ?

Reply:
This would require significant modifications to the current setup. Like updating `angular.json`, adding `webpack.config.js` to inject Environment Variables Dynamically, Unless there are other specific requirements